templates/ceph-storage.j2.j2

```diff
@@ -14,6 +14,7 @@ network_config:
     default: true
 {% endraw %}
 
+{% if manila_enabled %}
```

Comment:
Please submit a separate PR for manila related changes as jetpack is not yet supporting it for OSP17